src/common/SPELLS/PALADIN.js

```diff
@@ -23,6 +23,12 @@ export default {
     icon: 'spell_holy_righteousfury',
     manaCost: 600,
   },
+  JUDGMENT_CAST_ALT_2: {
```

Comment:
this name is a bit confusing, Judgment cast alternative 2? Please add a note explaining what this spellID is used for.